clients/java/src/main/java/io/camunda/client/impl/CamundaClientEnvironmentVariables.java

```diff
@@ -52,6 +52,9 @@ public final class CamundaClientEnvironmentVariables {
   public static final String OAUTH_ENV_CONNECT_TIMEOUT = "CAMUNDA_AUTH_CONNECT_TIMEOUT";
   public static final String OAUTH_ENV_READ_TIMEOUT = "CAMUNDA_AUTH_READ_TIMEOUT";
 
+  public static final String ENTRA_ENV_CERTIFICATE_PATH = "ENTRA_CERTIFICATE_PATH";
```

Comment:
🔧  I would align this with the other variables in this class i.e. `OAUTH_ENV_SSL_CLIENT_CERT_PATH`